tests/Unit/Elliptic/DiscontinuousGalerkin/Test_ImposeBoundaryConditions.cpp

```diff
@@ -0,0 +1,487 @@
+        std::move(bdry_vars), std::move(bdry_normal_dot_fluxes),
+        std::move(bdry_other_data), std::move(exterior_bdry_normal_dot_fluxes),
+        std::move(exterior_bdry_other_data), std::move(exterior_bdry_vars),
+        std::move(mortar_history), std::move(mortar_next_temporal_ids),
+        std::move(mortar_meshes), std::move(mortar_sizes));
+  }
+  ();
+
+  using MockRuntimeSystem = ActionTesting::MockRuntimeSystem<Metavariables>;
+  using MockDistributedObjectsTag =
+      typename MockRuntimeSystem::template MockDistributedObjectsTag<
+          my_component>;
+  typename MockRuntimeSystem::TupleOfMockDistributedObjects dist_objects{};
+  tuples::get<MockDistributedObjectsTag>(dist_objects)
+      .emplace(self_id, std::move(start_box));
+
+  ActionTesting::MockRuntimeSystem<Metavariables> runner{
+      {NumericalFlux{}}, std::move(dist_objects)};
+
+  using initial_databox_type = db::compute_databox_type<tmpl::append<
+      db::AddSimpleTags<
+          TemporalId, Tags::Next<TemporalId>, Tags::Mesh<2>, Tags::Element<2>,
+          Tags::ElementMap<2>, bdry_vars_tag,
+          bdry_normal_dot_fluxes_tag<flux_comm_types>, bdry_other_data_tag,
+          exterior_bdry_normal_dot_fluxes_tag<flux_comm_types>,
+          exterior_bdry_other_data_tag, exterior_bdry_vars_tag,
+          mortar_data_tag<flux_comm_types>, mortar_next_temporal_ids_tag,
+          mortar_meshes_tag, mortar_sizes_tag>,
+      compute_items<my_component>>>;
+
+  runner.template next_action<my_component>(self_id);
+
+  CHECK(runner.template is_ready<my_component>(self_id));
+
+  // Check that BC's were indeed applied.
+  const auto& exterior_vars = db::get<exterior_bdry_vars_tag>(
+      runner.template algorithms<my_component>()
+          .at(self_id)
+          .template get_databox<initial_databox_type>());
+
+  db::item_type<exterior_bdry_vars_tag> expected_vars{};
+  for (const auto& direction : external_directions) {
+    expected_vars[direction].initialize(3);
+  }
+  get<field_tag>(expected_vars[Direction<2>::lower_eta()]) =
+      Scalar<DataVector>({{{1., 2., 3.}}});
+  get<field_tag>(expected_vars[Direction<2>::upper_xi()]) =
+      Scalar<DataVector>({{{4., 5., 6.}}});
+  CHECK(exterior_vars == expected_vars);
+
+  // ReceiveDataForFluxes
+  runner.template next_action<my_component>(self_id);
+  const auto& self_box = runner.template algorithms<my_component>()
+                             .at(self_id)
+                             .template get_databox<initial_databox_type>();
+
+  auto mortar_history = serialize_and_deserialize(
+      db::get<mortar_data_tag<flux_comm_types>>(self_box));
+  CHECK(mortar_history.size() == 2);
+  const auto check_mortar = [&mortar_history](
+      const std::pair<Direction<2>, ElementId<2>>& mortar_id,
+      const Scalar<DataVector>& local_flux,
+      const Scalar<DataVector>& remote_flux,
+      const Scalar<DataVector>& local_other,
```

Comment:
What does the `other` refer to? some other non-flux/non-normal quantity? What is it used for?

Reply:
there is an `OtherData` tag which the test numerical flux defined at the beginning of this file packages into `ExtraData`